db.c

```diff
@@ -1193,7 +1193,7 @@ zval *php_mongo_runcommand(zval *zmongoclient, mongo_read_preference *read_prefe
 		zval_ptr_dtor(&cursor);
 		return NULL;
 	}
```

Comment:
If we compare this (`php_mongo_runcommand()`) with the `php_mongo_collection_findone()` code above, we're calling `php_mongo_handle_error()` before loading the current element. Effectively, the `$err` check will be ignored and we'll only handle errors if the error flag bit is set on the cursor. Is this an oversight?

The code path in `php_mongo_collection_findone()` (`php_mongo_handle_error()` after `php_mongocursor_load_current_element()`) looks correct.